pkg/bff/auth/testdata/invalid_token.txt

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImVkOGE0ODcyLWUwNDItNGU2NS04ZDQ3LTFkZDkzYTUwNTI0ZSJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImlhdCI6MTUxNjIzOTAyMn0.N0aJnJEE8lselYyqIQWupEvw9SBlc2tQ4YFqH6ACaxpnGz2nIbxU4PWP-E-rJ949a1GU24laozNHzyqNpZKrGvzHDeYdS8HcCZLIhgeU7bM1JWpXmMEmmPk-oi9EywVsbJo1Ll9dPcDIGPM_9OTc0MvgKEjLajCH8gA74wRal5QinKSR4VQIJkZ1AVG26SU77Qj8tHynOjoi14KHUghTh9zXc7Vb-mXQo002jTFgbmLl43qOsd7U6BfCTjVAWBoGhKMO3K9jyeeYIWqC8CTOB1eQQMT5F6qsn0t1R392F5tK-KPNmGZAXfvN9D6uVqauUTRgSPqq5rGb4a4DazhQ-w
```

Comment:
I created this token on jwt.io with a generated RSA key pair - it won't match Auth0's public keys which is why it's invalid.